Expensify/App
# Prevent tags for showing in sidebar for edited messages

### Details
Follow up to https://github.com/Expensify/Web-Expensify/pull/31834. Uses `message.text` instead of `message.html` so that we don't show tags in the sidebar when you edit a message

**HELD** until https://github.com/Expensify/Web-Expensify/pull/31834 is live. 
### Fixed Issues

$ https://github.com/Expensify/App/issues/4702

### Tests / QA Steps

1. Navigate to a conversation
2. Send a message
3. Edit the message and add a markdown to any of the words
4. Confirm the LHN shows the message preview without any HTML elements

### Tested On

- [ ] Android -- couldn't get android working locally, will tinker with it. 

### Screenshots


#### Web

https://user-images.githubusercontent.com/16747822/131931731-c04faf9c-146f-4419-bced-a0f7277ee936.mov

#### Mobile Web

https://user-images.githubusercontent.com/16747822/131931802-ce66b80c-4164-40b8-872b-4086c498e8b5.mov

#### Desktop

https://user-images.githubusercontent.com/16747822/131931824-58814d33-d1b7-4f39-b4de-0740c4064fad.mov

#### iOS

https://user-images.githubusercontent.com/16747822/131931842-98cf9648-f612-45bb-8b92-eadcaeaf06e3.mov

#### Android
